provisionersdk/proto/provisioner.proto

```diff
@@ -195,6 +195,7 @@ message Script {
 message Devcontainer {
 	string workspace_folder = 1;
 	string config_path = 2;
+	string name = 3;
```

Comment:
@spikecurtis Do we need to bump the version again? Or is it fine if this makes it into the release cut?

Reply:
IMO we don't need a separate version bump as long as this gets into the release.